frontend/ui/elements/screensaver_menu.lua

```diff
@@ -109,14 +109,16 @@ return {
         end,
         callback = function()
             G_reader_settings:saveSetting("screensaver_type", "disable")
-        end
+        end,
+        separator = true
```

Comment:
(For info: you can let a comma at the end of the last item - it will avoid showing in diff when you add an item - like this `end,` as you had to add the comma.)